static/app/views/releases/detail/overview/index.tsx

```diff
@@ -52,6 +54,8 @@ import ReleaseStats from './releaseStats';
 import TotalCrashFreeUsers from './totalCrashFreeUsers';
 
 const RELEASE_PERIOD_KEY = 'release';
+const FRONTEND_PLATFORMS: string[] = [...frontend, ...mobile];
+const BACKEND_PLATFORMS: string[] = [...backend, ...serverless];
```

Comment:
Is this duplicate of the one in `static/app/components/discover/performanceCardTable.tsx`?

Reply:
ah it is, sending through as a prop instead 👍🏼